# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373252258 25200
#      Sun Jul 07 19:57:38 2013 -0700
# Node ID bfd119642f6a6443baaa3ff6160e865bc9d814ac
# Parent  9a316ec12b5fbb23e9d5c7a76d8c75155da5e12e
doc: Fix some spellings in image.texi.

* doc/interpreter/doccheck/aspell-octave.en.pws: Add multipage to Octave
dictionary.

* doc/interpreter/image.txi: Correct spelling of amount.

* scripts/image/imformats.m: Use @nospell macro around "isa".

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -619,16 +619,17 @@ mput
 mrdivide
 msgbox
 MSYS
 mtimes
 Multi
 multi
 multibyte
 multiline
+multipage
 multipledelimsasone
 MultiSelect
 mxArray
 myclass
 myfun
 nabla
 namespace
 NAMESPACE
diff --git a/doc/interpreter/image.txi b/doc/interpreter/image.txi
--- a/doc/interpreter/image.txi
+++ b/doc/interpreter/image.txi
@@ -74,17 +74,17 @@ reading it into Octave.  This is done us
 provides read access to many of the parameters stored in the header of the image
 file.
 
 @DOCSTRING(imfinfo)
 
 By default, Octave's image IO functions (@code{imread}, @code{imwrite},
 and @code{imfinfo}) use the @code{GraphicsMagick} library for their
 operations.  This means a vast number of image formats is supported
-but considering the large ammount of image formats in science and
+but considering the large amount of image formats in science and
 its commonly closed nature, it is impossible to have a library
 capable of reading them all.  Because of this, the function
 @code{imformats} keeps a configurable list of available formats,
 their extensions, and what functions should the image IO functions
 use.  This allows to expand Octave's image IO capabilities by
 creating functions aimed at acting on specific file formats.
 
 While it would be possible to call the extra functions directly,
diff --git a/scripts/image/imformats.m b/scripts/image/imformats.m
--- a/scripts/image/imformats.m
+++ b/scripts/image/imformats.m
@@ -31,17 +31,17 @@
 ## field @code{ext}.  It contains the following fields:
 ##
 ## @table @asis
 ## @item ext
 ## The name of the file format.  This may match the file extension but Octave
 ## will automatically detect the file format.
 ## @item description
 ## A long description of the file format.
-## @item isa
+## @item @nospell{isa}
 ## A function handle to confirm if a file is of the specified format.
 ## @item write
 ## A function handle to write if a file is of the specified format.
 ## @item read
 ## A function handle to open files the specified format.
 ## @item info
 ## A function handle to obtain image information of the specified format.
 ## @item alpha
